cypress/support/uploadfile

```diff
@@ -0,0 +1 @@
+hello this is a test file
```

Comment:
may be add this file to a mock folder not in support folder

Reply:
need some help for this + not sure is it necessary ?